practice_question6.rb

```diff
@@ -0,0 +1,8 @@
+arr1 = [1,2,3,"himanshu"]
+arr2 = arr1.each do |e|
+  if e.is_a? Numeric
+    e = (e.to_i * 2).to_s
+  else e.to_s
+  end
```

Comment:
You can use ternary operator as well. And what's the need to assign value in `e` variable?